CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Add filter field
 - Convert OData dates to DSS format
+- Add individual user credentials preset
```

Comment:
and renamed SAP/ODATA login parameter to Service account

Reply:
done in [aaed917](https://github.com/dataiku/dss-plugin-sap-odata/pull/9/commits/aaed917b0a321eee724651010f752fddf7f088c4)

Reply:
I disagree your honor

Reply:
The changelog should contains 
```suggestion
- Add individual user credentials preset and renamed SAP/ODATA login parameter to Service account
```

Reply:
oops [b3909c3](https://github.com/dataiku/dss-plugin-sap-odata/pull/9/commits/b3909c3501237807c0006d0a3756f8d686154133)